okhttp/src/main/kotlin/okhttp3/internal/tls/TrustManagerOverride.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.internal.tls
+
+import javax.net.ssl.HostnameVerifier
+import javax.net.ssl.X509TrustManager
+
+open class TrustManagerOverride(
```

Comment:
should this just be a hostname since our configuration is just on or off?

Reply:
Dumbing it down to hostname for now.  FWIW a lot of this complexity comes from me thinking ahead to far to something more complicated e.g. setting up intranet certificate policies based on wildcards.  Removing all that for now.